agent/src/main.rs

```diff
@@ -1,18 +1,25 @@
+mod http_server;
 mod lister;
 mod options_parser;
 mod watcher;
-mod http_server;
+mod configuration_wrapper;
 
 use std::process;
 use std::thread;
 
+
 #[tokio::main]
 async fn main() {
     // Object configuration will be used to do config.get_host / config.get_port
```

Comment:
Maybe u can remove this comment if you implement the configuration wrapper

Reply:
until we aren't using it, there is no point in deleting the comment